__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const app = require("../app-files/app");
+const seed = require("../db/seeds/seed");
+const request = require('supertest')
+const db = require("../db/connection");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed({ articleData, commentData, topicData, userData });
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  test("GET: 200 responds with an array of topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const {topics} = response.body;
+        expect(topics).toHaveLength(3);
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+  test('ERROR: 400 when given an invalid pathway', ()=>{
```

Comment:
This bad path test should really be in a separate describe block, because this doesn't just apply to `api/topics` it will catch any bad path.
Good test though 👍